cal/cal/templates/home_logged_in.html

```diff
@@ -26,6 +26,13 @@
         }
       });
     });
+    function runFullSync() {
```

Comment:
Do you think you could this stuff in Angular instead? All of the other javascript in `home_logged_in.html` is copy pasted stuff from auth documentation, and we're trying to avoid putting arbitrary logic in the html (and usually when we do, it's in `calendaranalytics.js` - that's where the legacy javascript is). Some things you might want to check out:

- `LoggedInController` which is currently unused but would be perfectly suited for Angularizing this code
- `ng-init` for injecting data like you have using Django templating
- `$location` is the Angular way of changing the window location. We don't want to use pure javascript in this case. [Docs](https://docs.angularjs.org/guide/$location)

Thanks! Excited for this :D